src/composables/near.js

```diff
@@ -12,47 +12,28 @@ import {
 export const useMemes = () => {
   // memes starts as an empty array
   const memes = ref([]);
```

Comment:
question (blocking): do you need reactivity here? Are you updating any values after rendering it?